### PR TITLE
fix(nav): change the bottom nav in the mobile AP-675

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

changed the bottom menu in the mobile

**Which issue(s) this PR fixes** 🔨

AP-675

### CODE REVIEW

**Comment by KemoPaw** [comment]
Screen Shot 2022-02-14 at 5 23 28 PM
Screen Shot 2022-02-14 at 5 23 28 PM


Functions well, noticed that on the UI/UX example the gradient for the colors is from top --> bottom, but on this ticket it shows top left --> bottom right.

File: components/views/navigation/mobile/nav/Nav.less [hunk 1]
```diff
@@ -1,13 +1,29 @@
 #mobile-nav {
   width: 100vw;
   height: @mobile-nav-size;
-  background: @black;
+  background: linear-gradient(
+      163.91deg,
```
Comment (KemoPaw) [inline components/views/navigation/mobile/nav/Nav.less:5]: Make line 5 --> '180deg'

**Review by KemoPaw — COMMENTED** [review]
Let me know if these may help!

File: components/views/navigation/mobile/nav/Nav.less (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidchan1219) [reply components/views/navigation/mobile/nav/Nav.less:5]: changed

File: components/views/navigation/mobile/nav/Nav.less [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+      rgba(48, 52, 71, 0.5) 0%,
+      rgba(42, 54, 75, 0) 100%
+    ),
+    linear-gradient(
+      163.91deg,
+      rgba(28, 39, 58, 0.9) 0%,
+      rgba(0, 0, 0, 0.09) 100%
+    ),
+    linear-gradient(
+      359.95deg,
```
[the remark this answers is not in the document]
Comment (davidchan1219) [reply components/views/navigation/mobile/nav/Nav.less:15]: changed

File: components/views/navigation/mobile/nav/Nav.less [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
   width: 100vw;
   height: @mobile-nav-size;
-  background: @black;
+  background: linear-gradient(
+      163.91deg,
+      rgba(48, 52, 71, 0.5) 0%,
+      rgba(42, 54, 75, 0) 100%
+    ),
+    linear-gradient(
+      163.91deg,
```
[the remark this answers is not in the document]
Comment (davidchan1219) [reply components/views/navigation/mobile/nav/Nav.less:10]: changed

**Comment by stavares843** [comment]
@davidchan1219 

from liz

the bottom bar should always be active, the only time it's not is when in chat or in video calling mode. So basically if the user is chatting in any way like through dm's or group chat, or in a video call, etc. it's not displayed (cause normally the chat bar is there, or in the video, the buttons are more center of the screen or hidden)


let me know if you have any questions about it


atm the bar gets dismissed when you click on any of the icons

**Review by stavares843 — CHANGES_REQUESTED** [review]
added comment

**Comment by davidchan1219** [comment]
@stavares843 Just fixed the above issue.

**Comment by stavares843** [comment]
@davidchan1219 

the design was good how it was before the last commit

image


the only thing that needed modification was this

''the bottom bar should always be active, the only time it's not is when in chat or in video calling mode. So basically if the user is chatting in any way like through dm's or group chat, or in a video call, etc. it's not displayed (cause normally the chat bar is there, or in the video, the buttons are more center of the screen or hidden)''

**Comment by davidchan1219** [comment]
@stavares843 Now the nav bar always exists except chat page. I am not sure what is wrong. Could you tell me that in details?

**Comment by stavares843** [comment]
is possible to have the same design that had here

image


and having the nav bar always appearing except the chat page?

**Comment by phillsatellite** [comment]
While testing I found something I don't think Sara mentioned yet, but the bottom nav has some transparency to it, verified with Liz that the Nav bar should not have any transparency 
IMG_4654

**Comment by davidchan1219** [comment]
@phillsatellite fixed the nav opacity problem

**Comment by stavares843** [comment]
i agree with @phillsatellite

**Review by phillsatellite — APPROVED** [review]

**Review by genie-magic — APPROVED** [review]

**Review by Jekrimo — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**